open_academy/models/session.py

```diff
@@ -1,14 +1,42 @@
-from odoo import models, fields
+from odoo import models, fields, api
 
 
 class Session(models.Model):
     _name = 'session'
     _description = 'Session'
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today)
+    is_active = fields.Boolean(default=True)
```

Comment:
This is a special odoo record, keep it as 
```suggestion
    active = fields.Boolean(default=True)
```